app/assets/javascripts/govuk_publishing_components/lib/govspeak/magna-charta.js

```diff
@@ -118,7 +119,7 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     tr.classList.add('mc-tr')
 
     var output = ''
-    var allTheTHs = this.$table.querySelectorAll('th')
+    var allTheTHs = this.$table.querySelectorAll('thead th')
```

Comment:
I feel like this variable name is now a lie, would it be a pain to make it accurate to help future devs understand it?

Reply:
🍰 👍